src/components/MessagesList/MessagesGroup/Message/MessagePart/MessageBody.vue

```diff
@@ -258,6 +270,32 @@ export default {
 			return this.isInCall && this.pollsStore.isNewPoll(this.message.messageParameters.object.id)
 		},
 
+		isCallEndedMessage() {
+			return this.message.systemMessage === 'call_ended' || this.message.systemMessage === 'call_ended_everyone'
+		},
+
+		conversation() {
+			return this.$store.getters.conversation(this.message.token)
+		},
+
+		isTemporaryConversation() {
```

Comment:
`hasRetentionPeriod` name might explain it better. It's not the same as `isTemporary` for messages in the same file